assets/src/domain/eventEditor/ui/tickets/ticketsList/actionsMenu/ticketMainMenuItem/CopyTicket.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useCallback } from 'react';
+import { pick } from 'ramda';
+import { __ } from '@wordpress/i18n';
+
+import { Copy } from '@application/ui/layout/entityActionsMenu/entityMenuItems';
+import { useTicketContext } from '@edtrHooks/index';
+import { useTicketItem } from '@edtrServices/apollo/queries';
+import { useTicketMutator } from '@edtrServices/apollo/mutations';
+import { useRelations } from '@appServices/apollo/relations';
+
+const CopyTicket: React.FC = (props) => {
+	const { id } = useTicketContext();
+	const ticket = useTicketItem({ id });
+	const { createEntity } = useTicketMutator();
+	const { getRelations } = useRelations();
+	const newTicket = pick(
+		[
+			'description',
+			'endDate',
+			'isDefault',
+			'isExpired',
+			'isFree',
+			'isOnSale',
+			'isPending',
+			'isRequired',
+			'isSoldOut',
+			'isTaxable',
+			'isTrashed',
```

Comment:
These fields are not in mutation input. You should use either
```ts
import { TICKET_INPUT_FIELDS } from '@sharedEntities/tickets/predicates/ticketFields'
```

or `isTicketInputField` predicate from `@sharedEntities/tickets/predicates/selectionPredicates`

Reply:
May be similar change needs in COPY DATE. Please see `DatetimeBaseInput` from `@edtrServices/apollo/mutations/datetimes/types`